config/vite.json

```diff
@@ -1,17 +1,20 @@
 {
   "all": {
     "sourceCodeDir": "app/javascript",
-    "additionalEntrypoints": ["~/{icons,images}/**/*", "~/styles/*.scss"],
+    "additionalEntrypoints": ["~/{fonts,icons,images}/**/*", "~/styles/*.scss"],
```

Comment:
Isn't the static `~/styles/*.scss` a possibly breaking change from the previous parsing of `config/themes.yml`?